.gitignore

```diff
@@ -27,17 +27,14 @@ coverage
 *.sln
 *.sw?
 
-# venvs
-venv
-
 # pycache
 __pycache__
 
 # secrets
-envs.env
+.env
 
 # database
 mariadb
 
-# config test
-romm
```

Comment:
I think ``romm`` folder should stay because is the folder I use to mock the ``library``, ``config.yml`` and other stuff to test while developing

Reply:
Ah cool I'll add a comment explaining what it's for and why it should stay